.github/workflows/build-and-test.yaml

```diff
@@ -20,4 +20,9 @@ jobs:
       - name: test
         #TODO Remove this when we have tests
         run: yarn test --passWithNoTests
-
+      - name: Vulnerability Scan
+        uses: IABTechLab/uid2-shared-actions/actions/vulnerability_scan_filesystem@v2
+        with:
+          scan_severity: HIGH,CRITICAL
```

Comment:
Instead of specifying it here, can you help update the default value https://github.com/IABTechLab/uid2-shared-actions/blob/main/actions/vulnerability_scan_filesystem/action.yaml#L11 to be `HIGH,CRITICAL` please?